src/api/v1beta1/properties.go

```diff
@@ -323,8 +389,13 @@ func (dk *DynaKube) ImmutableOneAgentImage() string {
 		tag = truncatedVersion
 	}
 
```

Comment:
we should take the chance to rename this to just `OneAgentImage()` as it's not related to the immutable image at all - different naming used for activegate image as well, `CustomActiveGateImage()`

Reply:
done